java/src/jmri/InstanceManager.java

```diff
@@ -739,7 +681,7 @@ static public void setConfigureManager(ConfigureManager p) {
     //
     static public void setConsistManager(ConsistManager p) {
         store(p, ConsistManager.class);
-        getDefault().pcs.firePropertyChange(CONSIST_MANAGER, null, null);
+        getDefault().pcs.firePropertyChange("consistmanager", null, null);
```

Comment:
These notification services have been migrated into `store(...)` already, which will fire a property change for "default-ConsistManager", so I think this line can be removed entirely.

Reply:
I think CAT is still looking for the "consistmanager" string. I'll check.

Reply:
Is OK, will remove.